### ISSUE TITLE
Could some additional instructions be provided?

### ISSUE THREAD

**Issue by zimler** [osqzss/bladeGPS#7]
Hi!
I  was able to build the gps-sdr-sim using Visual Studio, but I'm struggling to get this to build. I must be missing a small step. Any additional instructions would be very much appreciated, thanks!

**Comment by osqzss** [comment]
You can find the instructions to build the bladeRF library from the source with Visual Studio 2013 Express for Windows Desktop in the Nuand wiki page:
https://github.com/Nuand/bladeRF/wiki/Getting-Started%3A-Windows

Assume you already downloaded pthread and libusb files and successfully built the bladeRF library.

In order to build bladeGPS, you need to add the paths to the following folders in Configuration Properties -> C/C++ -> General -> Additional Include Directoris:
1. “pthreads-w32-2-9-1-release/Pre-built.2/include” for pthread.h
2. “bladeRF/include” for libbladeRF.h

You also need to add the paths to the following folders in Configuration Properties -> Linker -> General -> Additional Library Directories:
1. “pthreads-w32-2-9-1-release/Pre-built.2/lib/x64” for pthreadVC2.lib
2. “bladeRF/x64” for bladeRF.lib

For the link command, specify the name of the additional libraries in Configuration Properties -> Linker -> Input -> Additional Dependencies:
1. pthreadVC2.lib
2. bladeRF.lib

Basically that's it! Now you should be able to build the code. In order to execute bladeGPS, you need the following DLLs in the same folder:
1. bladeRF.dll
2. libusb-1.0.dll
3. pthreadVC2.dll

Enjoy ;)

**Comment by osqzss** [comment]
Or you can try [the latest release](https://github.com/osqzss/bladeGPS/releases/tag/v0.2) for Windows.

**Comment by zimler** [comment]
Wow! Thank you for taking the time to put those instructions together and then compiling it. I hope to have time tonight to try it out.

**Comment by zimler** [comment]
the precompiled release is working well, thank you! I will try the compiling another time.
